plugins/dell-kestrel/fu-dell-kestrel-hid-device.c

```diff
@@ -228,6 +224,7 @@ fu_dell_kestrel_hid_device_write_firmware_pages(FuDellKestrelHidDevice *self,
 		g_autoptr(GByteArray) page_aligned = g_byte_array_new();
 		g_autoptr(FuChunk) page = NULL;
 		g_autoptr(GError) error_local = NULL;
+		guint page_ack_time = 0;
```

Comment:
doesn't this need to be `FU_DELL_KESTREL_HID_TIMEOUT`?

Reply:
ooh, that's right. a new patch is coming.